requirements/requirements.standalone.in

```diff
@@ -1 +0,0 @@
-pulp-container~=2.3.1
```

Comment:
If we are getting rid of requirements.standalone.in, can requirements.standalone.txt go as well?

Would need to be removed from dev/standalone/Dockerfile and the Makefile.

Or would/should that be a different pr?

Reply:
I left it in there just in case we want to add other standalone only dependencies. Not sure if it's the best thing to do or not

Reply:
I think it would  have to be updated in a bunch of different places

Reply:
Seems fine to me to leave it for now. We can remove it later if needed.